### PR TITLE
Provide: offset query param

### PR DESCRIPTION
**Pull request by omani**
as requested.

### CODE REVIEW

File: mongo.go [hunk 1]
```diff
@@ -201,9 +201,16 @@ func (m *Handler) Find(ctx context.Context, lookup *resource.Lookup, page, perPa
 	defer m.close(c)
 	var mItem mongoItem
 	query := c.Find(q).Sort(s...)
+
+	// Perform request the first time (to get total number of results without any filter applied)
+	iter := query.Iter()
+
```
Comment (rs) [inline mongo.go]: This should go back in place without the comment.

File: mongo.go [hunk 2]
```diff
@@ -213,10 +220,8 @@ func (m *Handler) Find(ctx context.Context, lookup *resource.Lookup, page, perPa
 		query.SetMaxTime(dur)
 	}
 	// Perform request
-	iter := query.Iter()
-	// Total is set to -1 because we have no easy way with Mongodb to to compute this value
-	// without performing two requests.
```
Comment (rs) [inline mongo.go]: Please don't remove this comment.

**Comment by omani** [comment]
I removed the artifacts.

**Comment by omani** [comment]
updated.

File: mongo.go [hunk 3]
```diff
@@ -214,9 +218,7 @@ func (m *Handler) Find(ctx context.Context, lookup *resource.Lookup, page, perPa
 	}
 	// Perform request
 	iter := query.Iter()
-	// Total is set to -1 because we have no easy way with Mongodb to to compute this value
-	// without performing two requests.
-	list := &resource.ItemList{Page: page, Total: -1, Items: []*resource.Item{}}
+	list := &resource.ItemList{Items: []*resource.Item{}}
```
Comment (rs) [inline mongo.go]: `Total` should still be `-1`

**Comment by rs** [comment]
Made required changed and merged into master.

**Pull request closed without merging**